src/lib/cfa/ntp_conf.rb

```diff
@@ -137,6 +157,8 @@ def last
       # @param [Record] record
       def <<(record)
         @augeas_tree.add(record.augeas[:key], record.augeas[:value])
+        # TODO: nasty workaround to survive multiline key with long comments
+        @augeas_tree.all_data.last[:multiline] = record.augeas[:multiline]
```

Comment:
Is it `:multiline` a real augeas key? Is it not possible to use `comment[]` for multiline comments?